### PR TITLE
Add metroupdate.biz bypass

### LINKED ISSUE #49: onsafelink.com
### Domain

https://onsafelink.com/m/zqryhCTHb

### Link

https://onsafelink.com/m/v1O1MCbs

### Version

0.0.0

### What browsers are you seeing the problem on?

Firefox

### What OS are you seeing the problem on?

Windows

### (Optional) Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by NotAProton**
<details> <summary>NOTICE</summary>
I dedicate any and all copyright interest in this software to the
public domain. I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors. I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.
</details>

Fixes: #49 

were they even trying lol

- [x] I made sure there are no unnecessary changes in the code*
- [x] Tested on Chromium- Browser OS
- [x] Tested on Firefox

\* indicates required

### CODE REVIEW

File: src/js/injection_script.js [hunk 1]
```diff
@@ -2325,6 +2325,7 @@ ensureDomLoaded(()=>{
 	})
 	domainBypass("techynroll.com", ()=>awaitElement("a#enablebtn", a=>safelyAssign(a.href)))
 	hrefBypass(/meostream\.com\/links\//,()=> ifElement("a#link",safelyNavigate))
+	domainBypass("metroupdate.biz", ()=>myFunction())
```
Comment (lem6ns) [inline src/js/injection_script.js:2328]: Ensure that the DOM has been loaded first.
```suggestion
	domainBypass("metroupdate.biz", ()=>ensureDomLoaded(()=>myFunction()))
```

**Review by lem6ns — COMMENTED** [review]
Ensure that the DOM has been loaded first, as it says in the review.

File: src/js/injection_script.js (hunk 1 above)
Comment (NotAProton) [reply src/js/injection_script.js:2328]: It is already in ensureDomLoaded
`	//Insertion point for domain-or-href-specific bypasses running after the DOM is loaded. Bypasses here will no longer need to call ensureDomLoaded.`

Comment (lem6ns) [reply src/js/injection_script.js:2328]: 👍

**Pull request merged**